src/EditorFeatures/TestUtilities/Diagnostics/TestDiagnosticAnalyzerDriver.cs

```diff
@@ -72,9 +72,19 @@ private async Task<IEnumerable<Diagnostic>> GetDiagnosticsAsync(DiagnosticAnalyz
             return allDiagnostics;
         }
 
+        private bool areSpansCompatible(TextSpan firstSpan, TextSpan secondSpan, bool shouldVerifyEntireSpan)
+        {
+            return shouldVerifyEntireSpan ? firstSpan.Equals(secondSpan) : firstSpan.IntersectsWith(secondSpan);
+        }
+
         public Task<IEnumerable<Diagnostic>> GetAllDiagnosticsAsync(DiagnosticAnalyzer workspaceAnalyzerOpt, Document document, TextSpan span)
         {
-            return GetDiagnosticsAsync(workspaceAnalyzerOpt, document, span, document.Project, getDocumentDiagnostics: true, getProjectDiagnostics: true);
+            return GetDiagnosticsAsync(workspaceAnalyzerOpt, document, span, document.Project, getDocumentDiagnostics: true, getProjectDiagnostics: true, false);
```

Comment:
>false [](start = 154, length = 5)

We always use named argument specification for literal arguments, just like the prior arguments.